h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostUtils.java

```diff
@@ -1,5 +1,7 @@
 package hex.tree.xgboost;
 
+import com.google.common.collect.ImmutableListMultimap;
```

Comment:
forgotten import? doesn't seem to be used

Reply:
resolved